frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -557,6 +577,7 @@ pub mod pallet {
 			+ Eq
 			+ Clone
 			+ sp_std::fmt::Debug
+			+ for<'a> sp_std::convert::TryFrom<&'a [IndexAssignmentOf<Self>], Error = sp_npos_elections::Error>
```

Comment:
I think it is totally fine to say any `CompactSolution ` is `TryFrom< IndexAssignmentOf >`, thus hinting at making this a bound on the `CompactSolution` itself rather than here.

Reply:
I see. I'd previously had trouble expressing that, but it turns out that `IndexAssignmentOf<C>` (as opposed to `IndexAssignmentOf<T>`) makes it easier to write down. 32ed1ec18.